src/run.js

```diff
@@ -93,7 +95,7 @@ export function createProvideWrapper({
     const stateChanged = !shallowEqual(state, newState);
     log('new state is:', stateChanged ? '(changed)' : 'NO CHANGE', newState);
     if (stateChanged) {
-      const wroteToRouter = syncToBrowser(state, newState);
+      const wroteToRouter = syncToBrowser(stringifyParams(state), stringifyParams(newState));
```

Comment:
`syncToBrowser` expects the two original state objects, if you stringify them you break it (this is why the tests are failing)

In order to use `stringifyParams` inside `buildo-state` I think we should remove or seriously re-think `syncToBrowser`.

An option for now could be exporting `stringifyParams` as a util to be used in the project.